e2e/fixtures/base/src/routes/index.tsx

```diff
@@ -0,0 +1,26 @@
+import type { JSX } from 'react'
+import type { TuonoProps } from 'tuono'
+
+interface IndexProps {
+  subtitle: string
+}
+
+export default function IndexPage({
+  isLoading,
+}: TuonoProps<IndexProps>): JSX.Element {
+  if (isLoading) {
+    return <h1>Loading...</h1>
+  }
+
+  return (
+    <div className="title-wrap">
+      <h1 className="title">
+        TU<span>O</span>NO
```

Comment:
This structure can be simplified. Maybe just asserting:
1. A static string
2. A string passed from the server

Reply:
I think point 2 is still missing. 
There is already a rust route in place: `e2e/fixtures/base/src/routes/index.rs`.
Could you please render the string in the page and check that it exists?

Reply:
Updated 👌